tests/resources/MST_mirror_2f_measurements.schema.yml

```diff
@@ -39,3 +39,21 @@ data:
         required: false
         units: cm
         type: double
+      - name: mirror_x
+        description: |-
+          Mirror panel position x.
+        required: false
+        units: cm
+        type: double
+      - name: mirror_y
+        description: |-
+          Mirror panel position y.
+        required: false
+        units: cm
+        type: double
+      - name: shape_type
```

Comment:
Same comment as above, maybe we can/should come up with a friendlier way to represent the shape